app/uk.gov.hmrc.helptosavestridefrontend/controllers/StrideController.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavestridefrontend.controllers
+
+import com.google.inject.Inject
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent}
+import uk.gov.hmrc.auth.core.AuthConnector
+import uk.gov.hmrc.helptosavestridefrontend.auth.StrideAuth
+import uk.gov.hmrc.helptosavestridefrontend.config.FrontendAppConfig
+import uk.gov.hmrc.helptosavestridefrontend.util.toFuture
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import uk.gov.hmrc.helptosavestridefrontend.views
+
+class StrideController @Inject() (val authConnector: AuthConnector,
+                                  val messagesApi:   MessagesApi)(implicit val frontendAppConfig: FrontendAppConfig)
```

Comment:
if we are ok to rename the service as above, then this comes `HelpToSaveAdminController` or something like that...